testing/simapp/app.go

```diff
@@ -370,6 +372,9 @@ func NewSimApp(
 		appCodec, keys[ibcexported.StoreKey], app.GetSubspace(ibcexported.ModuleName), ibctm.NewConsensusHost(app.StakingKeeper), app.UpgradeKeeper, scopedIBCKeeper, authtypes.NewModuleAddress(govtypes.ModuleName).String(),
 	)
 
+	// Setup packet server to call on Eureka tests
+	app.PacketServer = packetserverkeeper.NewKeeper(appCodec, app.IBCKeeper.ChannelKeeper, app.IBCKeeper.ClientKeeper)
```

Comment:
Needed this so that we can call into these functions from the TestChain struct

Reply:
I suppose we could also always create the PacketServer.Keeper struct on the fly using the IBCKeeper fields...